docs/source/daml/resource-management/index.rst

```diff
@@ -0,0 +1,11 @@
+.. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Resource Management in Daml Application Design
+##############################################
```

Comment:
@remyhaemmerle-da Not something that needs to be addressed right now, but this possibly a good area in which you and your team can contribute with known pitfalls in designing and implementing smart contracts in Daml.